### PR TITLE
Profile picture size issue of theme 2019

### PR DESCRIPTION
**Pull request by mibhumi**
-> Solved the issue of profile picture size. In which Size of profile image is increased.
Issue ref: [https://github.com/rtCamp/rtmedia-buddypress-profile-picture/issues/14](https://github.com/rtCamp/rtmedia-buddypress-profile-picture/issues/14)
->In `theme: 2019` has a issue in profile picture size while cropping the image to set profile picture. Issue is solved using overridden the class of css which was implementing.

### CODE REVIEW

File: app/assets/css/sass/_rtm.scss [hunk 1]
```diff
@@ -547,3 +547,24 @@ input.imgedit-submit-btn {
...
+	width: 0;
+}
+
+#buddypress div#item-header-cover-image .user-nicename {
+	margin: 0 0 5px;
+}
+
+#buddypress #item-header-content {
+	padding-left: 37px;
+}
```
Comment (surajkrsingh) [inline app/assets/css/sass/_rtm.scss]: Add new line at end

Comment (mibhumi) [reply app/assets/css/sass/_rtm.scss]: You are checking on old commit. Please check my post commit

**Comment by mibhumi** [comment]
yeah I did self testing

**Comment by kapilpaul** [comment]
@pooja-muchandikar please check if the issue still exits with new versions. if the issue still exits then test with this PR.

**Comment by pooja-muchandikar** [comment]
@kapilpaul 

The issue/add-on reference mentioned in PR description seems to be fixed and released with add-on version 1.2.0. I think this PR is no longer valid.

https://github.com/rtCamp/rtmedia-buddypress-profile-picture/issues/14#issuecomment-615080628

**Comment by kapilpaul** [comment]
> @kapilpaul
> 
> The issue/add-on reference mentioned in PR description seems to be fixed and released with add-on version 1.2.0. I think this PR is no longer valid.
> 
> [rtCamp/rtmedia-buddypress-profile-picture#14 (comment)](https://github.com/rtCamp/rtmedia-buddypress-profile-picture/issues/14#issuecomment-615080628)

If the issue is not exists hence I am closing this PR.

**Pull request closed without merging**